test/Validate.Capability.cpp

```diff
@@ -40,18 +40,51 @@
 namespace {
 
 using spvtest::ScopedContext;
+using std::get;
 using std::make_pair;
 using std::pair;
 using std::string;
-using std::stringstream;
+using std::ostringstream;
 using std::tuple;
 using std::vector;
 using testing::Combine;
 using testing::Values;
 using testing::ValuesIn;
 
-using ValidateCapability =
-    spvtest::ValidateBase<tuple<string, pair<string, vector<string>>>>;
+// Parameter for validation test fixtures. It's a tuple so Combine() can be used
+// to produce it.  The first string is a capability name, the second an assembly
+// text.  See below methods for how they're interpreted.
+using CapTestParameter = tuple<string, pair<string, vector<string>>>;
+
+const string& Capability(const CapTestParameter& p) { return get<0>(p); }
```

Comment:
This is done to use the testing::Combine functionality of google test framework. The combine construct requires tuples as far as I understand.